frontends/web/src/routes/device/bitbox01/restore.tsx

```diff
@@ -137,7 +137,6 @@ class Restore extends Component<Props, State> {
           activeDialog && (
             <DialogLegacy
               title={t('backup.restore.title')}
-              disableEscape={isConfirming || isLoading}
```

Comment:
This is a different component DialogLegacy, where you might want to keep it or also apply the change of Dialog component

Reply:
good catch, reverted.